integration-tests/tests/src/tests/list.ts

```diff
@@ -871,8 +873,11 @@ describe("Lists", () => {
         function testAssignNull(name: string, expected: string) {
           //@ts-expect-error TYPEBUG: our List type-definition expects index accesses to be done with a number , should probably be extended.
           expect(() => (prim[name] = [null])).throws(Error, `Expected '${name}[0]' to be ${expected}, got null`);
+          // TODO: Length should equal 1 when this is fixed: https://github.com/realm/realm-js/issues/6359
+          //       (This is due to catching the above error within this write transaction.)
           //@ts-expect-error TYPEBUG: our List type-definition expects index accesses to be done with a number , should probably be extended.
-          expect(prim[name].length).equals(1);
```

Comment:
Instead of patching the test here, I wonder if it'd make more sense to separate this out into another test and skip it for now? And eventually link to that from #6359.

Reply:
Sounds good 👍 I'll add `skip` to 2 other tests that also rely on that issue being solved.